pinot-controller/src/main/java/com/linkedin/pinot/controller/api/ControllerRestApplication.java

```diff
@@ -106,6 +105,11 @@ protected void configureRouter(Router router) {
 
     attachRoutesForClass(router, PinotVersionRestletResource.class);
 
+    // SegmentCompletionProtocol implementation on the controller
```

Comment:
Will these paths start showing up on the Swagger UI? If so, it may better to figure out a way to disable them from Swagger UI, as these are purely internal protocols, leaving the Swagger UI door open could mess up the protocol.